src/components/_modal/pages/orderHistoryDataConsent/orderHistoryDataConsent.page.js

```diff
@@ -0,0 +1,43 @@
+import Title from "@/components/base/title";
+import Text from "@/components/base/text";
+import styles from "./OrderHistoryDataConsent.module.css";
+import Button from "@/components/base/button";
+import Top from "@/components/_modal/pages/base/top";
+import Translate from "@/components/base/translate/Translate";
+
+export function OrderHistoryDataConsent({ modal }) {
```

Comment:
enten kun kommentar 
en kommentar for komponenten og måske en 
@file: Omelet.js - this file describes how to make an omelet